docs/ruleset_functions/get_rmr_scc_window_wall_ratios.md

```diff
@@ -1,15 +1,15 @@
 
-## get_building_scc_window_wall_ratios
+## get_rmr_scc_window_wall_ratios
 
-Description: This function would determine window and envelope wall ratios for a building for residential, non-residential, mixed and semi-heated surface conditioning categories.  
+Description: This function would determine window and envelope wall ratios for a RMR for residential, non-residential, mixed and semi-heated surface conditioning categories.  
```

Comment:
Please refer to it as RulesetModelInstance, as the RMR may have more than one ASHRAE229.ruleset_model_instances